### PR TITLE
enhancement: reordering the Operator tests to make sure the disruptive tests are executed at the last place

### PR DESCRIPTION
**Pull request by CFSNM**
Related ticket: https://issues.redhat.com/browse/RHOAIENG-14200

### CODE REVIEW

**Comment by mattmahoneyrh** [comment]
Have there been Jekins runs for this PR?

**Comment by CFSNM** [comment]
@mattmahoneyrh running at this moment

**Review by mattmahoneyrh — DISMISSED** [review]

**Comment by VaishnaviHire** [comment]
/lgtm

The PR needs a rebase

File: ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0110__service_mesh.robot [hunk 1]
```diff
@@ -94,73 +72,6 @@ Suite Teardown
...
-    ${file_path}=    Set Variable    ./tasks/Resources/Files/
-    ${return_code}=    Run And Return Rc    oc apply -f ${file_path}smcp_template.yml
-    Should Be Equal As Integers    ${return_code}    0
-
-Delete Smcp
-    [Documentation]    Delete the ServiceMeshControlPlane already created
-    Run and Watch Command
-    ...    oc delete ServiceMeshControlPlane basic --namespace ${SERVICE_MESH_CR_NS} --force --grace-period=0
-
 Operator Deployment Should Be Ready
```
Comment (MarianMacik) [inline ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0110__service_mesh.robot:75]: This keyword should be removed as well.

Comment (CFSNM) [reply ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0110__service_mesh.robot:75]: fixed

**Review by mattmahoneyrh — APPROVED** [review]

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**